client/src/components/reader/Section/Label.js

```diff
@@ -18,11 +46,31 @@ export default class Label extends PureComponent {
     if (this.isPublished()) return null;
     if (!this.category()) return null;
 
+    const fixedLabelClass = classNames({
+      "section-category-label": true,
+      fixed: true,
+      hidden: !this.state.fixed
+    });
+
```

Comment:
This could be condensed and simplified by taking advantage of className's ability to mix arg types and some destructuring:

```
const hidden = this.state.fixed;
const fixedLabelClass = classNames("section-category-label", "fixed", { hidden })
```